hudi-utilities/src/main/java/org/apache/hudi/utilities/checkpointing/InitialCheckpointFromAnotherHoodieTimelineProvider.java

```diff
@@ -35,7 +35,7 @@
  */
 public class InitialCheckpointFromAnotherHoodieTimelineProvider extends InitialCheckPointProvider {
 
-  private HoodieTableMetaClient anotherDsHoodieMetaclient;
+  private HoodieTableMetaClient anotherDsHoodieMetaClient;
```

Comment:
What's this change about?

Reply:
@ssdong
normalize strictly in camel-case. A journey of a thousand miles begins with a single step.